### ISSUE TITLE
Episode title is displayed for a few seconds (top-right) but nothing plays

### ISSUE THREAD

**Issue by sumnerboy12** [CastagnaIT/plugin.video.netflix#699]
#### Your Environment
- Netflix add-on version: 1.4.1
- Operating system version/name: OSMC June 2020
- Device model:  Vero 4K

Used Operating system:
* [ ] Android
* [ ] iOS
* [*] Linux
* [ ] OSX
* [ ] Raspberry-Pi
* [ ] Windows

### Describe the bug

Can login and browse my lists, i.e. login is successful, but when I select something to play I see the episode title appear in the top-right for a few seconds and then it disappears and nothing happens. 

Tried re-installing from scratch, rebooting, and downgrading to v1.4.0 and v1.3.2 with no effect. 

Wondering if this is something to do with a dependency that may have been upgraded recently? OSMC received an update a few days but all other streaming addons are working (e.g. Disney+).

#### Expected behavior

Episode should start playing.

#### Actual behavior

Episode doesn't start playing.

#### Steps to reproduce the behavior

1. Navigate to a show in "My List"
2. Select an episode and click to watch
3.

### Debug log

The debug log can be found from this link:
https://controlc.com/80fad3c7

#### Installation
* [ ] I'm using other Netflix Repo 
* [ ] I'm using a different source

**Comment by CastagnaIT** [comment]
Changing add-on version is not useful because the problem not come from add-on,
i suspect a problem with InputStreamAdaptive but i am not sure,

i need that you do this:
1) from fresh device startup
1) play only a **single** netflix video that not work
2) then open the user data kodi folder, and go to netflix folder
should be a path similar to: ...\userdata\addon_data\plugin.video.netflix
in this folder you can find 2 files:
manifest.json
manifest.mpd

attach the log and the other two files here, by add ".txt" to the end of filename
(do not ask help to me in how access to the OSMC kodi folder, i do not have device with OSMC)

**Comment by sumnerboy12** [comment]
Thanks @CastagnaIT. I just tried again and am getting a different error now, when I try to select a profile on the first screen I am getting `Exception: KeyError` (logs attached). This is different to what I was seeing last night, but I will attach the files you requested in case they help.

NOTE: I have tried this on two different OSMC Vero4K devices (running the same versions) and am seeing the `KeyError` exception on both.

[manifest.mpd](https://privatebin.bbjones.nz/?bc2bca3f880f22ec#MGI7TB1LNA+qVZ/ZTNJE6sW5k4Onlke4k8twi/ejhCU=)
[manifest.json](https://privatebin.bbjones.nz/?7c1d0deda434123c#df9TQQtcwIhjUWDbUG86b/V/qkJ1fTxuiEf2t8us1YE=)
[kodi.log](https://privatebin.bbjones.nz/?cab404d84b822987#Ov/eUA0URlFeHm7hg2hfCDQt48+94PaHef+5yebOWa8=)
